client/src/components/DefaultSnackbar.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import { Snackbar, SnackbarContent } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+	font: { 
+		color: theme.palette.primary.light,
+	}
+}));
+
+const DefaultSnackBar = function ({
+  open,
+  onClose,
+  message = "Sorry! Something Went Wrong!",
+  duration = 6000,
+}) {
+  const classes = useStyles();
+  const [close, setClose] = useState(true);
+
+  return (
+    <Snackbar
+      open={close && !!open}
```

Comment:
Is the !! being used to coerce a truthy/falsy value to a Boolean here?

Reply:
Yes the open property only accepts Boolean values, but you can pass in for example a string and it will automatically get coerced for you.